src/service.ts

```diff
@@ -135,7 +135,7 @@ export async function getCustomer(id: string, token: string): Promise<moltin.Cus
 export async function updateCustomer(id: string, name: string, email: string, token: string): Promise<{ data: moltin.Customer }> {
   const moltin = MoltinGateway({ host: config.endpointURL, client_id: config.clientId });
   // @ts-ignore
-  const result = await moltin.Customers.Update(id, {type: 'customer', name, email, password: '',}, token);
+  const result = await moltin.Customers.Update(id, {type: 'customer', name, email,}, token);
```

Comment:
Can you please remove the trailing comma after `email`. Also (I know it wasn't part of your change), can you add two spaces inside the curly brackets like on the line 136. So make it look like: `{ type: 'customer', name, email }`